portfolio/src/main/webapp/html/projects.html

```diff
@@ -0,0 +1,81 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Tony Shen - Projects</title>
+        <link rel="stylesheet" href="../css/style.css">
+        <link rel="stylesheet" href="../css/projects.css">
+        <!--Google Font API-->
+        <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto&effect=anaglyph">
+        <!--Social Media Links--> 
+        <link rel="stylesheet" 
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <script src="script.js"></script>
+    </head> 
+    <body>      
+        <div class="meta-header">
+            <img class="logo" src="../images/profile.jpg"/>    
+            <div class="upper-area">
+                <div class="header">        
+                    <div class="title-section"> 
+                        <a class="font-effect-anaglyph title" href="index.html">Tony Shen</a>
+                    </div> 
+
+                    <div class="nav-section">  
+                        <a class="font-effect-anaglyph nav-links" href="projects.html">Projects</a>
+                        <a class="font-effect-anaglyph nav-links" 
+                            href="https://drive.google.com/file/d/1lSsqU0wnAMAjw4Yj3lWPOef8z43ro9un/view?usp=sharing">
+                            CV</a>
+                        <a class="font-effect-anaglyph nav-links" href="blog.html">Vlog</a>
+                        <a class="font-effect-anaglyph nav-links" href="images.html">Images</a>
+                    </div>    
+
+                    <div class="nav-section-mobile">
+                        <span class="font-effect-anaglyph nav-links">Menu</span>
+                        <div class="dropdown-links" >
+                            <a class="font-effect-anaglyph nav-links" href="projects.html">Projects</a>
+                            <a class="font-effect-anaglyph nav-links" href="https://drive.google.com/file/d/1lSsqU0wnAMAjw4Yj3lWPOef8z43ro9un/view?usp=sharing">CV</a>
+                            <a class="font-effect-anaglyph nav-links" href="blog.html">Vlog</a>
+                            <a class="font-effect-anaglyph nav-links" href="images.html">Images</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="meta-project">
+            <div class="project-section">
+                <div class="project-item">
+                    <h3>Open ASO</h3>
+                    <p>Salio is a chatbot that adpats to your facebook personalities.</p>
```

Comment:
Another example of test strings that should be cleaned up before submission.

Reply:
Got it